src/rvwmo.adoc

```diff
@@ -503,19 +503,23 @@ register(s) to destination register(s) as specified
 
 |CSRRW‡ |_rs1_, _csr_^*^ | _rd_, _csr_ | |^*^unless _rd_=`x0`
 
+5+| ‡ carries a dependency from _rs1_ to _csr_ and from _csr_ to _rd_
+
 |CSRRS‡ |_rs1_, _csr_ |_rd_ ^*^, _csr_ | |^*^unless _rs1_=`x0`
 
 |CSRRC‡ |_rs1_, _csr_  |_rd_ ^*^, _csr_ | |^*^unless _rs1_=`x0`
 
-5+| ‡ carries a dependency from _rs1_ to _csr_ and from _csr_ to _rd_
+5+| ‡ carries a dependency from _csr_ and _rs1_ to _csr_ and from _csr_ to _rd_
```

Comment:
Alternatively, it might be clearer to handle the `rs1=x0` caveat here, rather than with the asterisk: "carries a dependency from csr to rd, and, if rs1 != x0, from _csr_ and _rs1_ to _csr_"... WDYT?

Reply:
I agree the asterisk is wrong, and I think it's fine to just move it rather than handle it in the text.